src/image/p5.Image.js

```diff
@@ -828,4 +831,87 @@ p5.Image.prototype.save = function(filename, extension) {
   p5.prototype.saveCanvas(this.canvas, filename, extension);
 };
 
```

Comment:
we'll need some documentation for play(), pause() and some addition to p5.Image() and loadImage() before this can be merged.

Reply:
Oh yeah I know. I still have a lot to do. I will change it out of draft status when I think it might be near merging status.